doc/source/_static/js/docsearch.js

```diff
@@ -23,10 +23,13 @@ const searchInput = document.getElementById("search-input");
 searchInput.addEventListener("keydown", function (e) {
     if (e.code === "Enter") {
         var searchForm = document.getElementsByClassName("bd-search")[0];
-
         const text = searchInput.value;
-        searchForm.action = "https://docs.ray.io/en/latest/search.html?q=" + text;
 
+        const pageUrl = window.location.href
+        const res = pageUrl.split("/");
+        const version = (res.length <= 3) ? "latest" : res[4];
```

Comment:
what about "master"?

Reply:
@richardliaw  this catches weird stuff like "localhost" and defaults to "latest" in that case, otherwise extracts the proper version such as `releases-1.13.0` or `master`.